src/pybind/cephfs/cephfs.pyx

```diff
@@ -1541,3 +1589,172 @@ cdef class LibCephFS(object):
             raise make_ex(ret, "error in lseek")
 
         return ret      
+
+    def utime(self, path, times=None):
+        """
+        Set access and modification time for path
+
+        :param path: file path for which timestamps have to be changed
+        :param times: if times is not None, it must be a tuple (atime, mtime)
+        """
+
+        self.require_state("mounted")
+        path = cstr(path, 'path')
+        if times:
+            if not isinstance(times, tuple):
+                raise TypeError('times must be a tuple')
+            if not isinstance(times[0], (int, float)):
+                raise TypeError('atime should be an int')
+            if not isinstance(times[1], (int, float)):
+                raise TypeError('mtime should be an int')
```

Comment:
Why do you check whether times[0] and times[1] is of type float when you say it should be of type `int` in the TypeError message?

Reply:
ah! error message needs fixing.